Makefile

```diff
@@ -128,11 +128,12 @@ test-cleanup:			# Only needed if e2e tests are interrupted.
 	@for TYPE in sa role rolebinding configmap; do oc get $$TYPE -o name; done | grep -e -receiver | xargs --verbose --no-run-if-empty oc delete
 	@oc get ns -o name | grep clo- | xargs --verbose --no-run-if-empty oc delete
 
-generate-bundle: $(OPM)
+MANIFEST_VERSION?="4.6"
+generate-bundle: regenerate $(OPM)
 	mkdir -p bundle; \
-	pushd bundle; \
-	$(OPM) alpha bundle generate --directory ../manifests/$(OCP_VERSION)/ --package cluster-logging-operator --channels $(OCP_VERSION) --output-dir .; \
-	popd
+	$(OPM) alpha bundle generate --directory manifests/${MANIFEST_VERSION} --package cluster-logging-operator --channels ${MANIFEST_VERSION} --default ${MANIFEST_VERSION} --output-dir bundle/; \
+	find bundle/manifests/ -type f ! \( -name "cluster-logging*" -o -name "*crd.yaml" \) -delete
```

Comment:
Please add bundle validation also in the same target, or in separate target
`$(OPERATOR_SDK) bundle validate --verbose ./bundle`